include/glibby/spatial/kdTree.h

```diff
@@ -0,0 +1,178 @@
+#pragma once
+
+#include <vector>
+#include <map>
+#include <queue>
+#include "glibby/primitives/point.h"
```

Comment:
we always put stl includes after our own includes (they should also be sorted alphabetically)